### PR TITLE
Changing killswitch flags

### PR DESCRIPTION
**Pull request by kneeyo1**
This removes the automator_modifiable flag from the killswitch options we have.

Leaving it blank has these options pick up the default flag [here](https://github.com/getsentry/sentry/blob/master/src/sentry/options/manager.py#L84).

### CODE REVIEW

File: src/sentry/options/defaults.py [hunk 1]
```diff
@@ -803,36 +803,16 @@
...
-    type=Dict,
-    default={},
-    flags=FLAG_AUTOMATOR_MODIFIABLE,
-)
+register("store.load-shed-group-creation-projects", type=Any, default=[])
+register("store.load-shed-pipeline-projects", type=Any, default=[])
+register("store.load-shed-parsed-pipeline-projects", type=Any, default=[])
+register("store.load-shed-save-event-projects", type=Any, default=[])
+register("store.load-shed-process-event-projects", type=Any, default=[])
+register("store.load-shed-process-event-projects-gradual", type=Dict, default={})
```
Comment (fpacifici) [inline src/sentry/options/defaults.py:811]: This is not a killswitch.

**Review by fpacifici — CHANGES_REQUESTED** [review]

File: src/sentry/options/defaults.py (hunk 1 above)
Comment (kneeyo1) [reply src/sentry/options/defaults.py:811]: oh right, this is a rollout flag

Comment (kneeyo1) [reply src/sentry/options/defaults.py:811]: my b

**Comment by kneeyo1** [comment]
We will be supporting killswitches in the future, closing this PR

**Pull request closed without merging**